### PR TITLE
Document getUnlockIdentifier()

### PR DESCRIPTION
**Pull request by pirapira**
because when I saw this function first, I couldn't tell the roles of the two address arguments of this function.

### CODE REVIEW

File: raiden_contracts/contracts/TokenNetwork.sol [hunk 1]
```diff
@@ -1086,6 +1086,16 @@ contract TokenNetwork is Utils {
         }
     }
 
+    /// @dev Get the hash of the channel identifier and the participant
+    /// addresses (whose orderings matter). The hash might be useful for
```
Comment (loredanacirstea) [inline raiden_contracts/contracts/TokenNetwork.sol]: `matter` -> `matters`

File: raiden_contracts/contracts/TokenNetwork.sol [hunk 2]
```diff
@@ -1086,6 +1086,16 @@ contract TokenNetwork is Utils {
...
+    /// looking up an UnlockData in
```
Comment (loredanacirstea) [inline raiden_contracts/contracts/TokenNetwork.sol]: `looking up an UnlockData in` -> `looking up the appropriate UnlockData for the participant`

File: raiden_contracts/contracts/TokenNetwork.sol (hunk 1 above)
Comment (pirapira) [reply raiden_contracts/contracts/TokenNetwork.sol]: will change into `whose ordering matters`.

File: raiden_contracts/contracts/TokenNetwork.sol (hunk 2 above)
Comment (pirapira) [reply raiden_contracts/contracts/TokenNetwork.sol]: will change into `... useful for the partner to look up the appropriate UnlockData to claim.`

**Pull request merged**